app/src/main/java/com/cornellappdev/transit/ui/components/MenuItem.kt

```diff
@@ -29,31 +29,45 @@ import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
  * @param sublabel The sublabel for the item
  */
 @Composable
-fun MenuItem(icon: ImageVector, label: String, sublabel: String, onClick: () -> Unit) {
+fun MenuItem(type: PlaceType, label: String, sublabel: String, onClick: () -> Unit) {
     Row(
         Modifier
             .fillMaxWidth()
+            .padding(horizontal = 16.dp, vertical = 6.dp)
             .clickable(onClick = onClick),
         verticalAlignment = Alignment.CenterVertically
     ) {
-        Icon(
-            icon,
-            contentDescription = "Place",
-            modifier = Modifier.padding(end = 20.dp),
-            tint = TransitBlue
-        )
+        if (type == PlaceType.APPLE_PLACE) {
+            Image(
+                painterResource(R.drawable.location_pin_gray),
+                contentDescription = "Place",
+                modifier = Modifier.padding(end = 20.dp),
+            )
+        } else {
+            Image(
+                painterResource(R.drawable.bus_stop_pin),
+                contentDescription = "Stop",
+                modifier = Modifier.padding(end = 20.dp),
+            )
+        }
         Column() {
             Text(
                 text = label,
                 fontSize = 14.sp,
                 color = PrimaryText,
-                fontFamily = sfProDisplayFamily
+                fontFamily = sfProDisplayFamily,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+                lineHeight = 16.sp
```

Comment:
You remember the vertical spacing thing driving us literally insane? I found the solution buried super deep in a Medium article.

Apply this:

```
                style = TextStyle(
                    platformStyle = PlatformTextStyle(
                        includeFontPadding = false
                    )
                )
```
to each `Text()`. The result is text that is correctly NOT vertically padded:

![image](https://github.com/user-attachments/assets/3d120f13-2c0d-4cdb-8e91-0defab2c3ccd)

hehe i already did it on my end but im making u do it now :-)